src/concurrently.spec.ts

```diff
@@ -30,7 +28,7 @@ beforeEach(() => {
 });
 
 it('fails if commands is not an array', () => {
-    const bomb = () => create('foo' as any);
+    const bomb = () => create('foo' as never);
```

Comment:
> I reckon this change wasn't automated 😄

You're right 😄 This is due to the [no-explicit-any](https://typescript-eslint.io/rules/no-explicit-any/) rule which is now enabled.

> I think that if we get all of TypeScript's `strict` mode properties toggled on, this won't compile.
> But it can stay for now 🙂

Actually, I think `never` is a perfect fit for this case and it seems like it does compile (tested quickly by adding `"strict": true` to `tsconfig.json`)